### ISSUE TITLE
Add warning/notice or explicit error regarding args structure

### ISSUE THREAD

**Issue by iron3oxide** [microsoft/vscode-isort#148]
It is very easy to encounter issues that are hard to debug by simply setting the isort args via the settings panel and thus not properly separating them (it is more intuitive to add "--profile black" than it is to add "--profile", "black", even though the latter makes sense once you start thinking about it).

In my case this lead to the following error:
```
ERROR: Filename override is intended only for stream (-) sorting.
```
The command seemed okay though:
```
python -m isort - --profile black 
```
It was only after I found #64 that I realized what my mistake had been. Before that I tried reinstalling the extension and meddling with every extension setting I could find. 

These are the solutions I could think of:

Solution 1: add a warning/notice in the extension readme to inform users that this can lead to hard to debug errors.

Solution 2: raise an error if the argument contains a space

**Comment by eleanorjboyd** [comment]
Hello! Thank you for the suggestion, but since parsing for all the different types of shell is very difficult and error prone this concept is out of scope for our extension.
